challenge-04/README.md

```diff
@@ -75,7 +115,23 @@ mostrar quantos assentos ainda podem ser ocupados, com a frase:
 - Se couber somente mais uma pessoa, mostrar a palavra "pessoa" no retorno
 citado acima, no lugar de "pessoas".
 */
-?
+carro.addPessoas = function(numPessoas) {
+  var assentosDisp = carro.assentos - carro.quantidadePessoas
+  var qtdPessoas = assentosDisp === 1 ? 'pessoa!' : 'pessoas!';
+
+  if (numPessoas != undefined && carro.quantidadePessoas >= 5) {
```

Comment:
-1
Sempre que for fazer uma verificação de igualdade, utilize o operador de comparação estrito (_strict equal_), que testa também o tipo :)